modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/RowVersion.java

```diff
@@ -36,11 +37,16 @@ public class RowVersion implements Storable {
      * A 'timestamp' representing absense of a timestamp.
      */
     public static final Timestamp NULL_TIMESTAMP = new Timestamp(Long.MIN_VALUE, Long.MIN_VALUE);
+
     /**
      * Represents an absent partitionless link.
      */
     public static final long NULL_LINK = 0;
 
+    public static boolean isNullLink(long link) {
+        return PageIdUtils.pageIndex(link) == 0;
```

Comment:
```suggestion
        return PageIdUtils.pageIndex(link) == NULL_LINK;
```

Reply:
No, NULL_LINK is a link, but 0 page index is not a link. It's not even a "long" value.

Reply:
I can introduce additional constant if you wish

Reply:
You know what, I think there's a way to simplify the code in some places